dev/io.openliberty.microprofile.openapi.4.0.internal.services/test/io/openliberty/microprofile/openapi40/internal/services/impl/test/ConfigFieldProvider40ImplTest.java

```diff
@@ -44,6 +44,7 @@ public class ConfigFieldProvider40ImplTest {
                                                                              "setAllowNakedPathParameter",
                                                                              "getConfigValue",
                                                                              "getConfigValueMap",
+                                                                             "getConfigValues",
                                                                              "toSet",
```

Comment:
This is a new utility method which is used by other default methods on the interface to retrieve a config property as a collection. It isn't a method we need to capture the result of when we serialize the config so I've added it to the ignore list.